apps/nextjs-website/src/components/molecules/QuestionsAndAnswers/QuestionsAndAnswers.tsx

```diff
@@ -0,0 +1,112 @@
+'use client';
+import React, { useState } from 'react';
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Box,
+  Button,
+  Typography,
+  useTheme,
+} from '@mui/material';
+import { Compress, Expand, ExpandMore } from '@mui/icons-material';
+import { useTranslations } from 'next-intl';
+import { QuestionsAndAnswersItem } from '@/lib/types/webinar';
+import BlocksRendererClient from '../BlocksRendererClient/BlocksRendererClient';
+import EContainer from '@/editorialComponents/EContainer/EContainer';
+
+const MIN_QUESTIONS_TO_SHOW = 5;
+const MAX_QUESTIONS_TO_SHOW = 10;
```

Comment:
You should convert these constants to input properties (minQuestionsToShow and maxQuestionsToShow) with 5 and 10 as default values